mirgecom/artificial_viscosity.py

```diff
@@ -158,9 +160,11 @@ class _AVRTag:
 
 
 def av_laplacian_operator(discr, boundaries, fluid_state, alpha, gas_model=None,
-                          kappa=1., s0=-6., time=0, operator_states_quad=None,
-                          grad_cv=None, quadrature_tag=None, boundary_kwargs=None,
+                          kappa=1., s0=-6., time=0, quadrature_tag=DISCR_TAG_BASE,
+                          volume_dd=DD_VOLUME_ALL, boundary_kwargs=None,
```

Comment:
`quadrature_tag` and `volume_dd` both contain a discretization tag.

You could do `volume_dd=None` and then allow (with a deprecation warning) a `quadrature_tag` to be passed, otherwise raise.

(Here and elsewhere.)

Reply:
As we discussed elsewhere, the two discretization tags here are only sometimes redundant. I tried to remove those cases.